src/main/java/org/semantics/apigateway/model/DynTransformResponse.java

```diff
@@ -101,8 +107,15 @@ private Map<String, Object> processItem(Map<String, Object> item, OntologyConfig
                 }
             }
             if (responseMapping.getType() != null && item.containsKey(responseMapping.getType())) {
-                newItem.put("type", item.get(responseMapping.getType()));
+                if (config.getDatabase().equals("ontoportal")) {
+                    newItem.put("type", "class"); // ontoportal do the search only on classes for now

```

Comment:
Would it be possible to use the local identifier? 

`ResourceFactory.createResource(String.valueOf(responseMapping.getType())).getLocalName().toLowerCase());`

This would be more generic in the future.